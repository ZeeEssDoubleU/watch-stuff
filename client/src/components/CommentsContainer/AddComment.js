import React from "react";
import PropTypes from "prop-types";
import { Image, Form, TextArea } from "semantic-ui-react";

import "./AddComment.scss";

const AddComment = props => (
	<div className="add-comment">
		<Image
			className="user-image"
			src="http://via.placeholder.com/48x48"
			circular
		/>
		<Form>
			<TextArea autoHeight rows={1} placeholder="Add a public comment..." />
		</Form>
	</div>
);

AddComment.propTypes = {};

export default AddComment;
