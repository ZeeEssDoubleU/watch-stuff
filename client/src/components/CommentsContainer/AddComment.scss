@import '../../styles/shared.scss';

.add-comment {
   display: flex;
   align-items: center;
   margin: 16px 0;

   .user-image {
      width: $avatar-diameter;
      height: $avatar-diameter;
      margin-right: $avatar-margin;
      cursor: pointer;
   }

   form {
      flex: 1;
   }
}