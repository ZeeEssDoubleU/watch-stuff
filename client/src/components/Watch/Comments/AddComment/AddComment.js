import React from "react";
import PropTypes from "prop-types";

import "./AddComment.scss";

const AddComment = props => {
	return <div className="add-comment">Add comment here.</div>;
};

AddComment.propTypes = {};

export default AddComment;
