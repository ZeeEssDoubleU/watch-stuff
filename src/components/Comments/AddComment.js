import React from "react";
import { Image, Form, TextArea } from "semantic-ui-react";

import "./AddComment.scss";

const AddComment = props => (
	<div className="add-comment">
		<Image
			className="user-image"
			src="https://via.placeholder.com/48x48"
			circular
		/>
		<Form>
			<TextArea
				autoHeight
				rows={1}
				placeholder="Add a public comment..."
			/>
		</Form>
	</div>
);

export default AddComment;
