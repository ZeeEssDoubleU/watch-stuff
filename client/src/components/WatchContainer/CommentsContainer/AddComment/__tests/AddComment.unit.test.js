import React from "react";
import { shallow } from "enzyme";
import AddComment from "../AddComment";

describe("AddComment", () => {
	test("renders correctly", () => {
		const wrapper = shallow(<AddComment />);
		expect(wrapper).toMatchSnapshot();
	});
});
