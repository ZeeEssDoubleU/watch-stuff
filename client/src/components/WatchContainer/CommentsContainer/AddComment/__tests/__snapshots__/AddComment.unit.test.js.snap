// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComment renders correctly 1`] = `
<div
  className="add-comment"
>
  <Image
    as="img"
    circular={true}
    className="user-image"
    src="http://via.placeholder.com/48x48"
    ui={true}
  />
  <Form
    as="form"
  >
    <TextArea
      as="textarea"
      autoHeight={true}
      control={[Function]}
      placeholder="Add a public comment..."
      rows={1}
    />
  </Form>
</div>
`;
